secret_store/src/key_server_cluster/math.rs

```diff
@@ -28,12 +29,27 @@ pub struct EncryptedSecret {
 	pub encrypted_point: Public,
 }
 
-/// Generate random scalar
+/// Create zero scalar.
+#[cfg(test)]
+pub fn zero_scalar() -> Secret {
+	Secret::zero()
+}
+
+/// Convert hash to EC scalar (modulo curve order).
+pub fn to_scalar(hash: H256) -> Result<Secret, Error> {
+	let scalar: U256 = hash.into();
+	let scalar: H256 = (scalar % math::curve_order()).into();
+	let scalar = Secret::from_slice(&*scalar);
+	scalar.check_validity()?;
```

Comment:
any secret in  `0..curve_order` is valid though, right?

Reply:
yes. except for 0 itself